middleware/redirects.global.ts

```diff
@@ -14,6 +15,10 @@ const getFormatAddressRouteCondition = (cond: ReplaceRouteItemCondition, { addre
     cond,
     replaceRoute: ({ params, name, query }) => {
       const address = params[addressKey].toString()
+
+      if (!isAddress(address)) {
```

Comment:
Even though this its sub specific 

We can use isAddress function that checks based on the vm , iirc we already have jt

Reply:
This function `isAddress` works for both evm and sub.